src/borg/cache.py

```diff
@@ -943,10 +943,16 @@ def wipe_cache(self):
         self.cache_config.integrity["files"] = fd.integrity_data
         self.cache_config.manifest_id = ""
         self.cache_config._config.set("cache", "manifest", "")
+        if not self.cache_config._config.has_section("integrity"):
+            self.cache_config._config.add_section("integrity")
+        for file, integrity_data in self.cache_config.integrity.items():
+            self.cache_config._config.set("integrity", file, integrity_data)
+        self.cache_config._config.set("integrity", "manifest", "")
```

Comment:
Please add a comment above this line explaining why you do that.

Reply:
(so one does not need to search in git history to find that)

Reply:
Added